versions.props

```diff
@@ -34,6 +34,9 @@ org.apache.thrift:libthrift = 0.9.2
 org.assertj:assertj-* = 3.5.2
 org.clojure:clojure = 1.8.0
 org.codehaus.groovy:groovy-all = 2.4.4
+org.eclipse.jetty:* = 9.4.0.v20161208
+org.eclipse.jetty.http2:* = 9.4.0.v20161208
```

Comment:
see comment on versions.lock above, but I think we have to pin separate Jetty versions for `timelock-server` versus the `atlasdb-dropwizard-bundle`